data-validation-cataloging-pipeline/app.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+
+import aws_cdk as cdk
+import json
+from data_validation_cataloging_pipeline.data_validation_cataloging import DataValidationCatalogingStack
+
+app = cdk.App()
+
+env = "dev"
+
+with open("./ddk.json") as f:
+    config_file = json.load(f)
```

Comment:
Is there a reason you didn't use [Config](https://awslabs.github.io/aws-ddk/release/latest/api/core/stubs/aws_ddk_core.config.Config.html#aws_ddk_core.config.Config) class to read the `ddk.json`? Is there any feature we're missing?

Reply:
Awesome, thanks!